core/app/views/spree/order_mailer/cancel_email.html.erb

```diff
@@ -8,35 +8,20 @@
         <%= Spree.t('order_mailer.cancel_email.instructions') %>
       </p>
       <p>
-        <%= Spree.t('order_mailer.cancel_email.order_summary_canceled') %>
+        <%= Spree.t('order_mailer.cancel_email.order_summary_canceled', number: @order.number) %>
       </p>
-      <table>
-        <% @order.line_items.each do |item| %>
-          <tr>
-            <td><%= item.variant.sku %></td>
-            <td>
-              <%= raw(item.variant.product.name) %>
-              <%= raw(item.variant.options_text) -%>
-            </td>
-            <td>(<%=item.quantity%>) <%= Spree.t('at_symbol') %> <%= item.single_money %> = <%= item.display_amount %></td>
-          </tr>
-        <% end %>
+      <table class="container">
         <tr>
-          <td></td>
-          <td><%= Spree.t('order_mailer.cancel_email.subtotal') %></td>
-          <td><%= @order.display_item_total %></td>
-        </tr>
-        <% @order.adjustments.eligible.each do |adjustment| %>
-          <tr>
-            <td></td>
-            <td><%= raw(adjustment.label) %></td>
-            <td><%= adjustment.display_amount %></td>
-          </tr>
-        <% end %>
-        <tr>
-          <td></td>
-          <td><%= Spree.t('order_mailer.cancel_email.total') %></td>
-          <td><%= @order.display_total %></td>
+          <td class="wrapper last">
+            <table class="twelve columns">
+              <%= render collection: @order.line_items, partial: 'spree/shared/mailer_line_item', as: :line_item %>
+              <%= render 'spree/order_mailer/subtotal', order: @order %>
+              <% @order.adjustments.eligible.each do |adjustment| %>
+                <%= render 'spree/order_mailer/adjustment', adjustment: adjustment %>
```

Comment:
@damianlegawiec - I guess we should avoid rendering of partials like these in the templates as partial render takes much time than html code written in template itself.

@jitendra @priyank-gupta @Mafi88 - Any thoughts?